protos/feast/core/CoreService.proto

```diff
@@ -22,139 +22,49 @@ option go_package = "github.com/gojek/feast/protos/generated/go/feast/core";
 option java_outer_classname = "CoreServiceProto";
 option java_package = "feast.core";
 
-import "feast/specs/EntitySpec.proto";
-import "feast/specs/FeatureGroupSpec.proto";
-import "feast/specs/FeatureSpec.proto";
-import "feast/specs/StorageSpec.proto";
-import "google/protobuf/empty.proto";
-import "google/protobuf/timestamp.proto";
+import "feast/core/FeatureSet.proto";
 
 service CoreService {
-  // Get entities specified in request.
-  // This process returns a list of entity specs.
-  rpc GetEntities(CoreServiceTypes.GetEntitiesRequest) returns (CoreServiceTypes.GetEntitiesResponse);
-  // Get all entities
-  // This process returns a list of entity specs.
-  rpc ListEntities(google.protobuf.Empty) returns (CoreServiceTypes.ListEntitiesResponse);
```

Comment:
Are we not having a ListFeatureSets method?

Reply:
My latest proto has this. Will push.